.github/workflows/notebook-test.yml

```
@@ -70,4 +74,15 @@ jobs:
           key: ${{ hashFiles('scripts/nb-tester/requirements.txt') }}
 
       - name: Run tox
-        run: tox -- ${{ steps.all-changed-files.outputs.all_changed_files }}
+        shell: python
+        run: |
+          import subprocess
+          files = """${{ steps.all-changed-files.outputs.all_changed_files }}"""
+          args = ["tox", "--"] + files.split("\n") + ["--write"]
+          subprocess.run(args)
```

Otherwise you won't propagate the error code:

```suggestion
          subprocess.run(args, check=True)
```